### PR TITLE
Delete comment from visitor

### PR DESCRIPTION
**Pull request by Gropoid**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring

## :scroll: Description
As a user I can delete a previously existing comment

## :green_heart: How did you test it?
manually, with sponsor account

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [x] All tests passing

### CODE REVIEW

File: lib/profile/profile_screen.dart [hunk 1]
```diff
@@ -225,8 +229,65 @@ class _ProfileScreenState extends State<ProfileScreen> {
     }
   }
 
+  void _delete(Comment comment) async {
+    print("_deleteComment()");
```
Comment (blacroix) [inline lib/profile/profile_screen.dart]:
```suggestion
```

File: lib/profile/profile_screen.dart [hunk 2]
```diff
@@ -225,8 +229,65 @@ class _ProfileScreenState extends State<ProfileScreen> {
...
+    final user =
+        User.fromJson(jsonDecode(await storage.read(key: STORAGE_KEY_USER)));
+    final accessToken = await storage.read(key: STORAGE_KEY_ACCESS_TOKEN);
+    final event =
+        Event.fromJson(jsonDecode(await storage.read(key: STORAGE_KEY_EVENT)));
+    final response = await http.delete(
+        '${DotEnv().env[ENV_KEY_API_URL]}/${user.tenant}/events/${event.id}/visitors/$visitorId/comments/${comment.id}',
+        headers: {HttpHeaders.authorizationHeader: "Bearer $accessToken"});
+    if (response.statusCode == 204) {
+      print("\tsuccess");
```
Comment (blacroix) [inline lib/profile/profile_screen.dart]:
```suggestion
```

File: lib/profile/profile_screen.dart [hunk 3]
```diff
@@ -156,7 +156,7 @@ class _ProfileScreenState extends State<ProfileScreen> {
                                 ? 8
                                 : 0),
                         child: Padding(
-                          padding: EdgeInsets.all(16),
+                          padding: EdgeInsets.all(8),
```
Comment (blacroix) [inline lib/profile/profile_screen.dart]: Why 8? Other CardView have 16 spacing? Could we manage to keep 16?

File: lib/profile/profile_screen.dart [hunk 4]
```diff
@@ -225,8 +229,65 @@ class _ProfileScreenState extends State<ProfileScreen> {
...
+    final event =
+        Event.fromJson(jsonDecode(await storage.read(key: STORAGE_KEY_EVENT)));
+    final response = await http.delete(
+        '${DotEnv().env[ENV_KEY_API_URL]}/${user.tenant}/events/${event.id}/visitors/$visitorId/comments/${comment.id}',
+        headers: {HttpHeaders.authorizationHeader: "Bearer $accessToken"});
+    if (response.statusCode == 204) {
+      print("\tsuccess");
+      this.commentController.text = '';
+      FocusScope.of(context).requestFocus(FocusNode());
+      setState(() {
```
Comment (blacroix) [inline lib/profile/profile_screen.dart]: Could we only remove comment from user without do new `http` call?

File: lib/profile/profile_screen.dart (hunk 3 above)
Comment (Gropoid) [reply lib/profile/profile_screen.dart]: The thing is, in the card just above ("add comment"), you "cheated" by using a Stack so the 16 margin does not apply to the "Save" icon.
So, if I left this one at 16, the "PopupMenu" icon and the Save icon would be grossly misaligned

Comment (blacroix) [reply lib/profile/profile_screen.dart]: Then could we add margin to text so text from above cell is align?

File: lib/profile/profile_screen.dart (hunk 4 above)
Comment (blacroix) [reply lib/profile/profile_screen.dart]: We can update snapshot.data instead of creating a Future wrapper

**Pull request merged**